test/integ_tests/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job_mocks.py

```diff
@@ -20,9 +20,17 @@ def pre_run_inject(mock_utils):
             ]
         }
     ])
-    mocker.set_create_job_result({
-        "jobArn" : f"arn:aws:braket:{mocker.region_name}:000000:job/testJob"
-    })
+    if mocker.mock_level == 'ALL':
+        # All full mocking is done in us-west-2
+        mocker.set_create_job_result({
+            "jobArn" : f"arn:aws:braket:{mocker.region_name}:000000:job/testJob"
+        })
+    else:
+        # When running on QPU, the job arn is based on the device arn region, even when mocking.
+        mocker.set_create_job_side_effect([
+            { "jobArn" : f"arn:aws:braket:{mocker.region_name}:000000:job/testJob"},
+            { "jobArn" : f"arn:aws:braket:us-west-1:000000:job/testJob"}
+        ])
```

Comment:
Do we really need a branch here? Why not just always mock the second way? (I think I was getting errors when I was trying that, so I believe there's an answer, I just don't know it)

Reply:
Yes, that's the reason. I tried to add those comments to explain why it's different between the two ways of mocking. We could change the way we do that, or we can add this branch here.

Reply:
I think the reason the second mocking doesn't work for all mocking is because `AwsSession.copy_session(region)` doesn't create a mock in the proper region. I've isolated this issue down to `boto3.Session(region_name=new_region)` not doing anything with `region_name`. Can we update the wrapper to fix this?

Reply:
I thought it was because the all mocking wrapper always returns us-west-2: https://github.com/amazon-braket/amazon-braket-examples/blob/main/test/integ_tests/mock_utils.py#L174

No?

Reply:
Correct, though because of where the region was being pulled from, the specific line of code was [a few lines up](https://github.com/amazon-braket/amazon-braket-examples/blob/main/test/integ_tests/mock_utils.py#L142)